STM/src/test/java/org/jboss/stm/internal/proxy/LockManagerProxyUnitTest.java

```diff
@@ -67,8 +67,8 @@ public String toString ()
     {
         public ExtendedLockable ()
         {
-            _isState = new Double(1.234);
-            _isNotState = new Integer(5678);
+            _isState = Double.valueOf(1.234);
```

Comment:
You don't need to box the primitive and then assign it to a primitive, ie `_isState = 1.234;` is sufficient. Similarly where the same has been done elsewhere in this PR.